### DISCUSSION TITLE
Skip next cron scheduled job

### DISCUSSION THREAD

**Discussion by IT-VBFK** [quartznet/quartznet#1533] [Q&A]
Hi

Is there any possibillity to skip explicitly the next calculated cron scheduled job.

E.g. if you have a job with this cron trigger expression `0 0 15 ? * SUN *`, so the next scheduled job execution would be sunday at 3:00 PM.

How to I skip this next execution and take the next possible _after_ this time?

**Comment by lahma** [comment]
You could check inside the job if run is needed or attach a calendar for the trigger when configuring.  Calendar will be checked whether calculated next fire time is valid, if not, triggers calculates next possible.
